client/components/layout/main-layout/MainLayout.tsx

```diff
@@ -1,24 +1,28 @@
 import React from 'react';
 import { Outlet } from 'react-router-dom';
-import Header from './header/Header';
 import Footer from './footer/Footer';
```

Comment:
This can be removed as the Footer is not used in the current version of main layout.

Reply:
Do you mean the Footer component? The Footer is part of the structure for the MainLayout. It's totally fine to keep it there, even if it's still a work in progress.